src/libelektra/keymoremeta.c

```diff
@@ -20,6 +20,8 @@
 
 #ifdef HAVE_ERRNO_H
 #include <errno.h>
+#elif defined(__APPLE__)
```

Comment:
`HAVE_ERRNO_H` is defined on my setup. Is there a certain configuration where `HAVE_ERRNO_H` fails / is not being defined?